### PR TITLE
Frozen disk threshold monitor logging

### PR DESCRIPTION
**Pull request by henningandersen**
Frozen nodes are expected to be above 90% disk usage, therefore we
disable any logging from the disk threshold monitor until flood stage
level. This commit also disables marking frozen indices read-only since
if we did so we may never remove the marking again (and they are already
read-only).

### CODE REVIEW

**Comment by henningandersen** [comment]
Failure was fixed in e484faa9ccb7b10092f8c420d9cf78218678e778

**Comment by henningandersen** [comment]
@elasticmachine update branch

File: server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitor.java [hunk 1]
```diff
@@ -155,6 +162,10 @@ public void onNewInfo(ClusterInfo info) {
                 continue;
             }
 
+            if (isFrozenOnlyNode(routingNode)) {
```
Comment (dakrone) [inline server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitor.java:165]: Could you add a short comment here about why we skip frozen nodes?

**Review by dakrone — APPROVED** [review]
LGTM, left one super minor comment

File: server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitor.java (hunk 1 above)
Comment (henningandersen) [reply server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitor.java:165]: Added in ff5250f59aed0d961375d49cc761dc5093d8c61e

**Comment by henningandersen** [comment]
Failures fixed by #71777

@elasticmachine run elasticsearch-ci/1
@elasticmachine run elasticsearch-ci/bwc

**Pull request merged**